content/en/training/_index.html

```diff
@@ -0,0 +1,106 @@
+          <b>Introduction to Cloud Infrastructure Technologies</b>
+        </h5>
+        <p>Learn the fundamentals of building and managing cloud technologies directly from The Linux Foundation, the leader in open source.</p>
+        <br>
+        <a href="https://www.edx.org/course/introduction-to-cloud-infrastructure-technologies" target="_blank" class="button">Go to Course</a>
+      </center>
+    </div>
+    <div class="col-nav">
+      <center>
+        <h5>
+          <b>Fundamentals of Containers, Kubernetes, and Red Hat OpenShift</b>
+        </h5>
+        <p>Learn practical techniques for creating and deploying containers on a Kubernetes cluster using Red Hat® OpenShift Container Platform.</p>
+        <br>
+        <a href="https://www.edx.org/course/fundamentals-of-containers-kubernetes-and-red-hat" target="_blank" class="button">Go to Course</a>
+      </center>
+    </div>
+  </div>
+</section>
+
+<div style="padding-top: 100px; padding-bottom: 100px; background-color: #4c4c4c; color: #fff">
+  <div class="main-section" style="width: 66%; margin-left: 16.5%">
+    <center>
+      <h2>Learn with the Linux Foundation</h2>
+      <p>The Linux Foundation offers instructor-led and self-paced courses for all aspects of the Kuberenetes application development and operations lifecycle.</p>
+      <a href="https://training.linuxfoundation.org/training/course-catalog/?_sft_technology=kubernetes" target="_blank" class="button">See Courses</a>
+    </center>
+  </div>
+</div>
+
+<section>
+  <div class="main-section" style="padding-top: 100px; padding-bottom: 100px">
+    <center>
+        <h2>Get Kubernetes Certified</h2>
+    </center>
+    <div class="col-nav">
+      <center>
+        <h5>
+          <b>Certified Kubernetes Application Developer (CKAD)</b>
+        </h5>
+        <p>The Certified Kubernetes Application Developer exam certifies that users can design, build, configure, and expose cloud native applications for Kubernetes.</p>
+        <br>
+        <a href="https://training.linuxfoundation.org/certification/certified-kubernetes-application-developer-ckad/" target="_blank" class="button">Go to Certification</a>
+      </center>
+    </div>
+    <div class="col-nav">
+      <center>
+        <h5>
+          <b>Certified Kubernetes Administrator (CKA)</b>
+        </h5>
+        <p>The Certified Kubernetes Administrator (CKA) program provides assurance that CKAs have the skills, knowledge, and competency to perform the responsibilities of Kubernetes administrators.</p>
+        <br>
+        <a href="https://training.linuxfoundation.org/certification/certified-kubernetes-application-developer-ckad/" target="_blank" class="button">Go to Certification</a>
+      </center>
+    </div>
+  </div>
+</section>
+
+<div style="padding-top: 100px; padding-bottom: 100px; background-color: #cccccc;">
+  <div class="main-section" style="width: 66%; margin-left: 16.5%">
+    <center>
+      <h2>Kubernetes Training Partners</h2>
```

Comment:
Thought: This section could end the page on a bigger visual/mental note if we used the same JS-driven table from the Partners page here too. Thoughts?

Reply:
Agreed! 👍